logback-core/src/main/java/ch/qos/logback/core/rolling/helper/CompressionRunnable.java

```diff
@@ -20,16 +20,19 @@
   final String nameOfFile2Compress;
   final String nameOfCompressedFile;
   final String innerEntryName;
+  final boolean alwaysDelete;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2371885199/issues/source?bid=3222618&fileBranchId=3222682#l23)